lib/Frontend/ArgsToFrontendOptionsConverter.cpp

```diff
@@ -310,6 +310,40 @@ bool ArgsToFrontendOptionsConverter::convert(
                      "-enable-library-evolution");
   }
 
+  Opts.AllowNonResilientAccess = Args.hasArg(OPT_experimental_allow_non_resilient_access);
+  if (Opts.AllowNonResilientAccess) {
+    // Override the option to skip non-exportable decls.
+    if (Opts.SkipNonExportableDecls) {
+      Diags.diagnose(SourceLoc(), diag::warn_ignore_option_overriden_by,
+                     "-experimental-skip-non-exportable-decls",
+                     "-experimental-allow-non-resilient-access");
+      Opts.SkipNonExportableDecls = false;
+    }
+    // Overriden by enable-testing as it already bypasses resilience.
+    if (Opts.EnableTesting) {
+      Diags.diagnose(SourceLoc(), diag::warn_ignore_option_overriden_by,
+                     "-experimental-allow-non-resilient-access",
+                     "-enable-testing");
+      Opts.AllowNonResilientAccess = false;
+    }
+    // Overriden by enable-private-imports as it already bypasses resilience.
+    if (Opts.EnablePrivateImports) {
+      Diags.diagnose(SourceLoc(), diag::warn_ignore_option_overriden_by,
```

Comment:
Same feedback as above; `-enable-private-imports` and `-experimental-allow-non-resilient-access` seem compatible. Why make the developer pick one or the other?